src/main/java/uk/gov/pay/api/model/links/PaymentWithAllLinks.java

```diff
@@ -62,6 +66,15 @@ private PaymentWithAllLinks(String chargeId, long amount, PaymentState state, St
         this.links.addSelf(selfLink.toString());
         this.links.addKnownLinksValueOf(paymentConnectorResponseLinks);
     }
+    
+    private boolean isCapturePresent(List<PaymentConnectorResponseLink> paymentConnectorResponseLinks) {
+        for (PaymentConnectorResponseLink link : paymentConnectorResponseLinks) {
+            if(link.getRel().equals("capture")) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
We can do something like this:

```java
return paymentConnectorResponseLinks.stream().anyMatch(link -> "capture".equals(link.getRel()));
```

Then we probably don’t even need to have a separate `isCapturePresent` method and can just inline it.

(Please also put the constant first when comparing — `"capture".equals(link.getRel())` rather than `link.getRel().equals("capture")` — because it won’t throw an exception if `link.getRel()` returns `null`.)

Reply:
my bad :(